WordPress/Classes/Utility/BackgroundTasks/WeeklyRoundupBackgroundTask.swift

```diff
@@ -611,6 +619,10 @@ class WeeklyRoundupNotificationScheduler {
         dateComponents: DateComponents,
         completion: @escaping (Result<Void, Error>) -> Void) {
 
+        if NotificationFilteringService().shouldFilterWordPressNotifications() {
+            return
```

Comment:
As mentioned in the previous comment, I think this is more transparent from the code readers perspective: 

```suggestion
            notificationFilteringService.cancelAllPendingWordPressLocalNotifications()
            return
```

Reply:
Not relevant anymore. Lets leave it in `shouldFilterWordPressNotifications`